model/crystal.h

```diff
@@ -1025,15 +1043,41 @@ namespace dxtbx { namespace model {
       cell_volume_sd_ = 0;
     }
 
+    void set_recalc_unit_cell(const cctbx::uctbx::unit_cell &unit_cell) {
+      has_recalc_cell_ = true;
+      recalc_unit_cell_ = unit_cell;
+    }
+
+    cctbx::uctbx::unit_cell get_recalc_unit_cell() const {
+      return recalc_unit_cell_;
+    }
+
+    void set_recalc_cell_parameter_sd(const scitbx::af::small<double, 6> &unit_cell_sd)
```

Comment:
It seems inconsistent to have the name `set_recalc_unit_cell ` but `set_recalc_cell_parameter_sd` - i.e. one contains `unit_cell` and one just `cell`. I wonder if the more verbose `recalculated` would be better (given you've got the full word `parameter` which could also potentially be truncated to `param` if you were trying to save characters)?

Reply:
The naming here followed the existing `get_cell_parameter_sd`. I do see the issue though.

Shortening to `recalc` was suggested by @graeme-winter. I have no preference. Happy to keep the longer form if that seems better - I let you guys decide